src/main/java/com/example/persimmoncocktails/mapper/RestorePasswordMapper.java

```diff
@@ -0,0 +1,17 @@
+package com.example.persimmoncocktails.mapper;
+
+import com.example.persimmoncocktails.dtos.auth.RestorePasswordDataDto;
+import org.springframework.jdbc.core.RowMapper;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.time.LocalDateTime;
+
+public class RestorePasswordMapper  implements RowMapper<RestorePasswordDataDto> {
+    @Override
+    public RestorePasswordDataDto mapRow(ResultSet rs, int rowNum) throws SQLException {
+        LocalDateTime localDateTime = rs.getTimestamp("time").toLocalDateTime();
+        if(rs.wasNull()) localDateTime = null;
```

Comment:
I think, there is bug.
rs.wasNull is true if previous call to rs for data as null (in our case it's rs.getTimestamp("time")).
But if it's null, we would try to call .toLocalDateTime() on null value, that would cause NullPointerException

Reply:
Yes, you are right